groovy/postInit/chemistry/inorganic_chemistry/elements/d_block/group9/CobaltChain.groovy

```diff
@@ -2,19 +2,77 @@ import globals.Globals
 import static globals.CarbonGlobals.*
 
 ROASTER = recipemap('roaster')
+EBF = recipemap('electric_blast_furnace')
 PBF_RECIPES = recipemap("primitive_blast_furnace")
+FLUIDIZEDBR = recipemap('fluidized_bed_reactor')
+BR = recipemap('batch_reactor')
+ELECTROLYTIC_CELL = recipemap('electrolytic_cell')
+
+//ROAST RECOVERY OF As2O3
+//COBALTITE MODIFICATION
+mods.gregtech.electric_blast_furnace.removeByInput(120, [metaitem('dustCobaltite')], [fluid('oxygen') * 3000])
+
+
+EBF.recipeBuilder()
+        .inputs(ore('dustCobaltite'))
+        .fluidInputs(fluid('oxygen') * 9000)
+        .outputs(metaitem('dustImpureCobaltOxide'))
+        .chancedOutput(metaitem('dustArsenicTrioxide') * 5, 7500, 0)
+        .fluidOutputs(fluid('sulfur_dioxide') * 2000)
+        .EUt(Globals.voltAmps[1])
+        .blastFurnaceTemp(1023)
+        .duration(240)
+        .buildAndRegister()
+
+FLUIDIZEDBR.recipeBuilder()
+        .inputs(ore('dustCobaltite'))
+        .fluidInputs(fluid('oxygen') * 9000)
+        .outputs(metaitem('dustImpureCobaltOxide'))
+        .chancedOutput(metaitem('dustArsenicTrioxide') * 5, 9000, 0)
+        .fluidOutputs(fluid('sulfur_dioxide') * 2000)
+        .EUt(Globals.voltAmps[2])
+        .duration(120)
+        .buildAndRegister()
+
+BR.recipeBuilder()
+        .inputs(ore('dustImpureCobaltOxide'))
+        .fluidInputs(fluid('sulfuric_acid') * 1000)
+        .fluidOutputs(fluid('impure_cobalt_sulfate_solution') * 2000)
+        .EUt(Globals.voltAmps[1])
+        .duration(240)
+        .buildAndRegister()
+
+ELECTROLYTIC_CELL.recipeBuilder()
```

Comment:
(CoSO4)(H2O) -> Co + O + H2SO4, just output sulfuric acid instead of acidic wastewater.